GiEnJul/Auth/HasScopeHandler.cs

```diff
@@ -0,0 +1,21 @@
+
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+
+namespace GiEnJul
```

Comment:
As the file is placed in the Auth folder, namespace should be GiEnJul.Auth
Also applies for HasScopeRequirement.cs

Reply:
added with a0b6dd1